src/targets/gpu/compile_gen.cpp

```diff
@@ -183,7 +184,10 @@ void generate_pointwise(cpp_generator& gg, const module& pm, const std::string&
 {
     module m = pm;
     run_passes(m,
-               {rewrite_quantization{}, eliminate_common_subexpression{}, dead_code_elimination{}});
+               {rewrite_quantization{},
+                eliminate_convert{},
+                eliminate_common_subexpression{},
```

Comment:
We should just use `optimize_module` here.

Reply:
Done.